catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -147,11 +156,8 @@ public boolean sourceStatus(String servicePID) {
           try {
             Configuration config = helper.getConfiguration(cs);
             if (config != null && config.getProperties().get("service.pid").equals(servicePID)) {
-              try {
-                return source.isAvailable();
-              } catch (Exception e) {
-                LOGGER.info("Couldn't get availability on source {}: {}", servicePID, e);
-              }
+                final String sourceId = source.getId();
+                return sourceIsAvailable(sourceId);
```

Comment:
Could we just make this `sourceIsAvailable(source.getId());`

Reply:
... or pull it up before the try so you can reuse that local variable in the log at line 163